src/passes/LocalSubtyping.cpp

```diff
@@ -152,7 +152,8 @@ struct LocalSubtyping : public WalkerPass<PostWalker<LocalSubtyping>> {
         // Remove non-nullability if we disallow that in locals.
         if (newType.isNonNullable()) {
           if (cannotBeNonNullable.count(i)) {
-            newType = Type(newType.getHeapType(), Nullable);
+            newType =
+              Type(newType.getHeapType(), Nullable, newType.getExactness());
```

Comment:
Perhaps it makes sense to add `getAsNullable()` or `getWithNullability(nullability)`? That would shorten such code, and be forward looking for when we add yet more things aside from shareability, nullability, exactness, etc....

Reply:
Actually it  could be `getAs(nullability)` which is nice and short.